scripts/authfix.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
We need a better name for this script file. Also it will be nice to have comment section here with the description what it doing and why/when it's needed.